### DISCUSSION TITLE
Single-window GUI: How to insert back a pane after moving it?

### DISCUSSION THREAD

**Discussion by veroandreo** [OSGeo/grass#2211] [Q&A] [answered]
I'm not sure if this is a bug or just me not understanding how the moving panes around works, so I post my question here. If a bug, we can move this to issues hopefully. 

I'm currently testing the single-window GUI from main branch and just to play I removed the Data Catalog from the column on the left where it is placed by default. Now, I do not find the way to insert it back to where it was. I get some kind of controls in my screen when I move the pane around holding the mouse left button, but these disappear when I drop it. I do not know how to reach them nor if they are supposed to be used for re-inserting removed panes. 

![image](https://user-images.githubusercontent.com/20075188/154167328-adc7ca44-e07b-4f9c-be5f-aa367f287c1e.png)

Can anyone explain how it works, please? :pray:

**Comment by veroandreo** [comment, marked as answer]
I just found it! :facepalm: Hovering the pane over those controls I get blue areas for indication of where to drop it! Cool! :smiley: 
Sorry for the noise!
![image](https://user-images.githubusercontent.com/20075188/154168117-dcb2d12f-fca8-437f-bef8-df318d0ae5ec.png)

**Reply by wenzeslaus** [reply]
Sounds great! We need post like this to help documenting everything!
